LICENSE.md

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (C) 2015-2017 MyEtherWallet LLC
+Copyright (C) 2017-2018 MyEllaWallet
```

Comment:
MyEtherWallet LLC has the original copyright. So I think we'd need to keep that. And this probably needs to changed to:

```
Original work Copyright (C) 2015-2017 MyEtherWallet LLC
Modified work Copyright (C) 2017-2018 The Ellaism Project
```

Reply:
Ah that was sloppy. Fixing